src/data/units.json

```diff
@@ -540,7 +540,7 @@
 	{
 		"id": 6,
 		"name": "Vehemoth",
-		"playable": false,
+		"playable": true,
```

Comment:
Planning to finish Vehemoth abilities next so hopefully it's ok to activate it!

Reply:
Not a problem. Go for it! :)